tools/src/uniolunisaar/adam/ds/petrinet/PetriNetExtensionHandler.java

```diff
@@ -245,6 +245,18 @@ public static void removeWeakFair(Transition t) {
         t.removeExtension(AdamExtensions.weakFair.name());
     }
 
+    public static boolean hasOrigID(Transition transition) {
```

Comment:
In commit [a0e12bd](https://github.com/adamtool/framework/commit/a0e12bdee2fbf1c10c69ea717e9abfecdfa0da49) I changed the type of the OrigID methods from place to node. Thus, you can now just use these methods for your transitions.